client/cypress/integration/homepage.spec.js

```diff
@@ -6,22 +7,30 @@ describe('Homepage', () => {
 
   it('contains a subheader for total donations', () => {
     cy.visit('/');
-    cy.contains('Number of meals that have been given to those in need thus far:');
+    cy.contains(
+      'Number of meals that have been given to those in need thus far:',
+    );
   });
 
   it('contains the total for donations', () => {
     cy.visit('/');
     cy.get('#total');
   });
 
-
   it('contains a subheader for total meals claimed', () => {
     cy.visit('/');
-    cy.contains('Number of meals that have been saved from waste thus far:');
+    cy.contains('Number of meals that have been donated thus far:');
```

Comment:
Personal opinion but I don't like the use of thus far for copy, it doesn't feel very trendy

Reply:
Same. I didn't write it, but we can do a big round of copy changes tomorrow.